### DISCUSSION TITLE
Welcome to gookme Discussions!

### DISCUSSION THREAD

**Discussion by LMaxence** [LMaxence/gookme#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect between users of Gookme. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

To get started, comment below with an introduction of yourself and tell us about how Gookme might have helped you, and how you use it :)

**Comment by NodeJSmith** [comment]
Just found this tool and it looks like it may be a good option for my team - only concern/question is with the name. It sounds quite close to a [slur](https://en.wikipedia.org/wiki/Gook) - do the maintainers have any concerns regarding this or have any plans to rename the tool?
